Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-biolinkml = ">=1.5.4"
+biolinkml = ">=1.5.10"
```

Comment:
this should be a separate PR

Reply:
Depends... BiolinkML 1.5.4 doesn't understand the strange partitioned mappings. I can't recall exactly which BiolinkML update handles them (Harold will know) but some of the mappings in the Publications are those refined mappings.  I can "dumb them down" to vanilla mappings, but then again, guess what? We are nearly ready (I hope)  to finalise the mappings PR, so... LOL.  

Actually, this Publications PR is a 'Draft'  PR. My expectation at this point is simply to clarify whether or not the  model is generally going in the correct direction. Most of your feedback (below) is well  aligned with that need, therefore, shall we turn a blind eye to the above BiolinkML version quirk for the moment? By the time this PR is officially ready to merge, I suspect that we'll want to update the main production PR accordingly (Harold may be content to issue a PR for the BiolinkML update in isolation given that his iterations between 1.5.4. through 1.5.10 have fixed a number of important background issues...)